service/authorization/authorization.go

```diff
@@ -337,15 +373,8 @@ func (as *AuthorizationService) GetEntitlements(ctx context.Context, req *author
 		}
 		var attributeFqns []string
 		for _, attr := range listAttributeResp.GetAttributes() {
-			ns := attr.GetNamespace().GetName()
-			an := attr.GetName()
 			for _, val := range attr.GetValues() {
```

Comment:
can avoid this extra looping once https://github.com/opentdf/platform/pull/1050 is merged